docs/api/auto-updater.md

```diff
@@ -109,10 +109,12 @@ using this API.
 Restarts the app and installs the update after it has been downloaded. It
 should only be called after `update-downloaded` has been emitted.
 
-**Note:** `autoUpdater.quitAndInstall()` will close all application windows
+**Note 1:** `autoUpdater.quitAndInstall()` will close all application windows
 first and only emit `before-quit` event on `app` after that. This is different
 from the normal quit event sequence.
 
+**Note 2:** If you don't call `autoUpdater.quitAndInstall()` after `update-downloaded` event has been emitted then you will get the updated app on the next start/run of your app.
+
```

Comment:
Not clear that actually update will be installed on quit. i.e. yes, result is that user will get updated app on next launch, but for developer maybe important to understand when update will be installed.

Reply:
Speaking for Squirrel.Mac, the update is installed when the app quits. I _think_ that's also true of Squirrel.Windows, but I can't say for certain.

Reply:
Squirrel.Windows installs while the app is still running, quitAndInstall and windows basically just restarts the app from the squirrel trampoline, the install is already done

Reply:
unfortunately this doesn't work for background running apps. if I shutdown or restart the windows, on next bootup the app still holds the older version and the update cycle continues unless i explicitly "quit" the app.